### PR TITLE
feat: Livechat theming improvements

### PR DESCRIPTION
**Pull request by aleksandernsilva**
## Proposed changes (including videos or screenshots)

**Added two settings**
- `Livechat widget position on the screen`: Changes the widget position between left or right of the viewport
- `Livechat background`: Changes the message list background. Receives the same value as the CSS's background property.

**Added new properties to the `setTheme` method**
- `position`: Changes the widget position between left or right of the viewport
- `background`: Changes the message list background. Receives the same value as the CSS's background property.
- `guestBubbleBackgroundColor`: Changes the guest bubble background color
- `agentBubbleBackgroundColor`: Changes the agent bubble background color
- `hideGuestAvatar`: Hides the guest avatar
- `hideAgentAvatar`: Hides the agent avatar

ℹ️ **Tests for these theming improvements will be implemented in a separate task: [PROSVC-79](https://rocketchat.atlassian.net/browse/PROSVC-79)**

## Issue(s)

[PROSVC-51](https://rocketchat.atlassian.net/browse/PROSVC-51)
[PROSVC-50](https://rocketchat.atlassian.net/browse/PROSVC-50)

## Further comments

[PROSVC-51]: https://rocketchat.atlassian.net/browse/PROSVC-51?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[PROSVC-79]: https://rocketchat.atlassian.net/browse/PROSVC-79?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .changeset/fuzzy-vans-own.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+"@rocket.chat/meteor": patch
+"@rocket.chat/livechat": patch
```
Comment (MarcosSpessatto) [inline .changeset/fuzzy-vans-own.md:3]:
```suggestion
"@rocket.chat/meteor": minor
"@rocket.chat/livechat": minor
```

**Review by MarcosSpessatto — CHANGES_REQUESTED** [review]

File: packages/livechat/src/widget.ts [hunk 2]
```diff
@@ -225,6 +226,15 @@ const openWidget = () => {
 	emitCallback('chat-maximized');
 };
 
+const setWidgetPosition = (position: 'left' | 'right' = 'right') => {
+	if (!widget) {
```
Comment (MartinSchoeler) [inline packages/livechat/src/widget.ts:230]: Let's give a warn if the position is invalid, so the user knows that it did not work

**Review by MartinSchoeler — CHANGES_REQUESTED** [review]

File: packages/livechat/src/components/Screen/ScreenProvider.tsx [hunk 3]
```diff
@@ -50,13 +58,41 @@ export const ScreenContext = createContext<ScreenContextValue>({
...
+		dispatch,
+		config,
+		sound,
+		minimized = true,
+		undocked,
+		expanded = false,
+		alerts,
+		modal,
+		iframe,
+		...store
```
[the remark this answers is not in the document]
Comment (aleksandernsilva) [reply packages/livechat/src/components/Screen/ScreenProvider.tsx:71]: @MartinSchoeler this is not a spread. I'm actually collecting all other properties into a `const` called `store` 🙃

File: apps/meteor/app/livechat/server/api/lib/livechat.ts [hunk 4]
```diff
@@ -176,6 +176,8 @@ export async function settings({ businessUnit = '' }: { businessUnit?: string }
 			color: initSettings.Livechat_title_color,
 			offlineTitle: initSettings.Livechat_offline_title,
 			offlineColor: initSettings.Livechat_offline_title_color,
+			position: initSettings.Livechat_widget_position || 'right',
+			background: initSettings.Livechat_background,
```
Comment (tassoevan) [inline apps/meteor/app/livechat/server/api/lib/livechat.ts:180]: `question` Why settings like `guestBubbleBackgroundColor` are not passed here?

**Review by tassoevan — CHANGES_REQUESTED** [review]

File: apps/meteor/app/livechat/server/api/lib/livechat.ts (hunk 4 above)
Comment (aleksandernsilva) [reply apps/meteor/app/livechat/server/api/lib/livechat.ts:180]: @tassoevan Not all features were added as WS settings. Some of them only exist in the widget API. There are plans to add these settings to the Livechat Appearance page in the future, but it's not in the scope for 6.7

File: apps/meteor/ee/app/livechat-enterprise/server/settings.ts [hunk 5]
```diff
@@ -194,6 +194,33 @@ export const createSettings = async (): Promise<void> => {
...
+			{ key: 'right', i18nLabel: 'Right' },
+		],
+		enterprise: true,
+		invalidValue: 'right',
+		modules: ['livechat-enterprise'],
+		enableQuery: omnichannelEnabledQuery,
+	});
+
+	await settingsRegistry.add('Livechat_background', '', {
+		type: 'string',
```
[the remark this answers is not in the document]
Comment (aleksandernsilva) [reply apps/meteor/ee/app/livechat-enterprise/server/settings.ts:214]: I added a description to this setting 🚀

**Review by tassoevan — APPROVED** [review]

**Review by KevLehman — APPROVED** [review]

**Review by MartinSchoeler — APPROVED** [review]

**Review by MarcosSpessatto — APPROVED** [review]

**Pull request merged**